rotkehlchen/chain/arbitrum_one/modules/umami/balances.py

```diff
@@ -0,0 +1,220 @@
+                                method_name='balanceOf',
+                                arguments=[address],
+                            ),
+                        ) for address in addresses_with_deposits
+                    ],
+                    call_order=call_order,
+                    calls_chunk_size=chunk_size,
+                )
+
+                if not results:
+                    log.error(f'Failed to query umami vault balances for vault {vault_contract.address}')  # noqa: E501
+                    continue
+
+                for idx, result in enumerate(results):
+                    user_address = addresses_with_deposits[idx]
+                    if (balance := vault_contract.decode(
+                        result=result,
+                        method_name='balanceOf',
+                        arguments=[user_address],
+                    )[0]) == 0:
+                        continue
+
+                    self._process_vault_balance(
+                        balance=balance,
+                        balances=balances,
+                        vault_token=vault_token,
+                        user_address=user_address,
+                    )
+            except RemoteError as e:
+                log.error(f'Failed to query umami balances for vault {vault_token.evm_address}: {e!s}')  # noqa: E501
+                continue
+
+        # get staked balances.
+        for user_address in addresses_with_deposits:
+            try:
+                results = self.evm_inquirer.multicall(
+                    calls=[
+                        (
+                           self.umami_masterchef_contract.address,
+                           self.umami_masterchef_contract.encode(
+                               method_name='userInfo',
+                               arguments=[idx, user_address],
+                           ),
+                        ) for idx in range(4)
+                    ],
+                    call_order=call_order,
+                    calls_chunk_size=3,  # etherscan throws an error for >3 chunk size.
+                )
+                if not results:
+                    log.error(f'Failed to query umami vault balances for address {user_address}')
+                    continue
+
+                for idx, result in enumerate(results):
+                    balance, _ = self.umami_masterchef_contract.decode(
+                        result=result,
+                        method_name='userInfo',
+                        arguments=[idx, user_address],
+                    )
+
+                    if balance == 0:
+                        continue
+
+                    gm_vault_token_address = gm_vault_addresses[idx]
+                    gm_vault_token = GlobalDBHandler().get_evm_token(
+                        address=gm_vault_token_address,
+                        chain_id=self.evm_inquirer.chain_id,
+                    )
+                    if gm_vault_token is None:
```

Comment:
start using the `:=` operator

```suggestion
                    if (gm_vault_token := GlobalDBHandler().get_evm_token(
                        address=gm_vault_token_address,
                        chain_id=self.evm_inquirer.chain_id,
                    )) is None:
```